app/assets/javascripts/admin/addon/models/user-field.js

```diff
@@ -19,6 +20,16 @@ export default class UserField extends RestModel {
   static fieldTypeById(id) {
     return this.fieldTypes().findBy("id", id);
   }
+
+  @tracked field_type;
+  @tracked editable;
+  @tracked show_on_profile;
+  @tracked show_on_user_card;
+  @tracked searchable;
+
+  get fieldTypeName() {
+    return UserField.fieldTypes().findBy("id", this.field_type).name;
```

Comment:
Prefer classic JS `find((fieldType) => fieldType.id === this.field_type)`, the dev-xp team is slowly removing these ember array extensions like findBy. Sorry for suggesting this in the past, I didn't realise

Reply:
Fixed. 👍